src/main/java/com/sobytylnik/Profile.java

```diff
@@ -18,48 +18,104 @@
     @Column(name = "surname")
     @Pattern(regexp="^[a-zA-Z]+")
     private String surname;
-    @Column(name = "age")
-    @Min(value = 1, message = "Age should not be less than 1")
-    @Max(value = 120, message = "Age should not be greater than 120")
-    private int age;
+    @Column(name = "location")
+    private String location;
+    @Column(name = "birthday")
+    private String birthday;
```

Comment:
Use `LocalDate` instead. Newest Hibernate should support this data type.
But if you encounter some problems, this article could help: https://thoughts-on-java.org/persist-localdate-localdatetime-jpa/